app/javascript/mastodon/features/account/components/account_note.js

```diff
@@ -162,7 +193,28 @@ class AccountNote extends ImmutablePureComponent {
           onKeyDown={this.handleKeyDown}
           onBlur={this.handleBlur}
           ref={this.setTextareaRef}
+          style={{ display: editable ? 'block' : 'none' }}
         />
+        {value ?
+          <div
+            role='button'
+            tabIndex={0}
+            className='account__header__account-note__show'
+            onClick={this.setEditable}
+            dangerouslySetInnerHTML={{ __html: emojifiedValue }}
+            style={{ display: editable ? 'none' : 'block' }}
+          />
+          :
+          <div
+            role='button'
+            tabIndex={0}
+            className='account__header__account-note__empty'
+            style={{ display: editable ? 'none' : 'block' }}
+            onClick={this.setEditable}
+          >
+            {intl.formatMessage(messages.placeholder)}
+          </div>
```

Comment:
Instead of having their visibility changed, maybe they could appear/disappear depending on `editable`?

Reply:
Yes, `{editable ? <Textarea>... : <div>...}` is better.
I don't know why I used CSS controls, but no fatal reason there

Reply:
I think if you do avoid rendering the Textarea when not needed, you should be able to use `autoFocus` and not use your sleep function from above.